f8a_worker/workers/bookkeeper.py

```diff
@@ -67,7 +67,9 @@ def execute(self, arguments):
         :param arguments: dictionary with task arguments
         :return: {}, results
         """
-        self._strict_assert(arguments.get('external_request_id'))
+        if arguments['data'].get('api-name') == 'stack_analyses':
```

Comment:
what is this fix for?

Reply:
@prashbnair It will fix error msg in worker in case someone hits our api without request_id

Reply:
Isnt this about external_request_id?